config/webpack.config.js

```diff
@@ -119,7 +119,14 @@ var config = {
       }
     }),
     new webpack.optimize.CommonsChunkPlugin({
-      name: 'vendor'
+      name: 'vendor',
+      minChunks(module) {
+        if (module.resource && (/^.*\.(css|scss)$/).test(module.resource)) {
```

Comment:
why not only .js files? or css is excluded?

Reply:
tbh I don't have a strong opinion about it.
Maybe it would be better to vendor all types of chunks?

I agree that if we decide to restrict so we should restrict to js only.